MSY/Day9/크레인 인형뽑기 게임.js

```diff
@@ -0,0 +1,25 @@
+function solution(board, moves) {
+    moves = moves.map(mv => mv - 1);
+    
+    let stack = [];
+    let cnt = 0;
+    
+    for (let i of moves) {
+        for (let row of board) {
+            if (row[i] != 0) {
```

Comment:
```suggestion
            if (row[i] !== 0) {
```
이렇게 하면 더 안전하게 타입까지 비교가 가능합니다!